### PR TITLE
Con 2766 fav list

### PR DESCRIPTION
**Pull request by sznowicki**

### CODE REVIEW

File: components/NavDrawer/components/Item/style.js [hunk 1]
```diff
@@ -26,12 +26,31 @@ const icon = css({
...
   padding: `${variables.gap.big + 2}px 0`,
   fontSize: '0.875rem',
   fontWeight: 500,
+};
+
+const label = css(labelBaseStyle).toString();
+
+const labelWithIndicator = css({
+  ...labelBaseStyle,
+  display: 'flex',
```
Comment (CarinaHoffmann) [inline components/NavDrawer/components/Item/style.js]: display:flex not needed here

File: components/NavDrawer/components/Item/style.js [hunk 2]
```diff
@@ -26,12 +26,31 @@ const icon = css({
...
+  justifyContent: 'space-between',
+  paddingRight: 32,
```
Comment (CarinaHoffmann) [inline components/NavDrawer/components/Item/style.js]: Can be replaced with varibales.gab.xbig

File: components/NavDrawer/components/Item/style.js (hunk 1 above)
Comment (sznowicki) [reply components/NavDrawer/components/Item/style.js]: True. Removed.

File: components/NavDrawer/components/Item/style.js (hunk 2 above)
Comment (sznowicki) [reply components/NavDrawer/components/Item/style.js]: Done.

**Pull request merged**